### PR TITLE
Removing the deploy folder from the lira repository

### PR DESCRIPTION
**Pull request by rhiananthony**
### Purpose
The purpose of this PR is to remove the deployment folder from the Lira repository since deployments will use the secondary-analysis-deploy repo going forward

- No issue is linked to this PR.
---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- No changes.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- No instructions.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -1,54 +0,0 @@
...
-10. Renders the Lira config file
-11. Deploys the lira config file with the caas key to the Kubernetes cluster
-12. Generates the lira deployment file
-13. Deploys the lira-deployment.yaml file to the Kubernetes cluster
-
-
-## New Deployments
-In addition to the above scripts there are several other items which need to be set up to deploy lira for the first time:
-
-### Create Service Accounts
```
Comment (rexwangcc) [inline deploy/README.md:29]: It's OK to remove this file, but I think we need to make sure these paragraphs eventually are covered by the documentation of the Lira on, say ReadtheDocs. @rhiananthony @samanehsan

**Review by rexwangcc — APPROVED** [review]

File: deploy/README.md (hunk 1 above)
Comment (rhiananthony) [reply deploy/README.md:29]: Ok, then I shouldn't merge this until that work is done. I'll do that next

Comment (samanehsan) [reply deploy/README.md:29]: How about adding a deployment section to the Lira readme that includes a link to the `secondary-analysis-deploy` repo?

**Comment by samanehsan** [comment]
LGTM @rhiananthony! As for documentation, I suggest adding a deployment section to the Lira README with a link to the secondary-analysis-deploy repo :octocat: 🌤

**Review by rexwangcc — CHANGES_REQUESTED** [review]
We might also want to remove the `.gitlab-ci.yml` file.

**Comment by samanehsan** [comment]
LGTM @rhiananthony! I think it's confusing to leave this in since we're now using the secondary-analysis-deploy repo :octocat:

**Pull request merged**